### PR TITLE
DIT-251 Added integration tests for create case

### PR DESCRIPTION
**Pull request by jumpertester**
Added integration tests for create case. This included setting up wiremock and test data.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -29,7 +29,13 @@ lazy val test = Seq(
   "org.pegdown" % "pegdown" % "1.6.0" % scope,
   "org.scalatest" %% "scalatest" % "3.0.4" % scope,
   "uk.gov.hmrc" %% "hmrctest" % "3.1.0" % scope,
-  "uk.gov.hmrc" %% "reactivemongo-test" % "3.0.0" % scope
+  "uk.gov.hmrc" %% "reactivemongo-test" % "3.0.0" % scope,
+  "org.scalatest" %% "scalatest" % "3.0.4" % scope,
+  "org.scalatestplus.play" %% "scalatestplus-play" % "2.0.1" % scope,
+  "org.mockito" % "mockito-core" % "2.11.0" % scope,
```
Comment (gokyo) [inline build.sbt]: some of these dependencies are defined already above

Comment (jumpertester) [reply build.sbt]: Re-ordered and removed duplicates.

File: build.sbt [hunk 2]
```diff
@@ -29,7 +29,13 @@ lazy val test = Seq(
...
+  "com.typesafe.play" %% "play-test" % PlayVersion.current % scope,
+  "org.scalaj" %% "scalaj-http" % "1.1.6" % scope,
+  "com.github.tomakehurst" % "wiremock" % "1.58" % scope
```
[the remark this answers is not in the document]
Comment (jumpertester) [reply build.sbt]: Updated the version of scalaj. Is this what you meant?

File: test/it/uk/gov/hmrc/component/CaseSpec.scala [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  import uk.gov.hmrc.bindingtariffclassification.model.JsonFormatters._
+
+  feature("Create Case") {
+
+    scenario("Create a new case") {
+
+      When("I create a new case")
+      val expectedCaseBody = Json.toJson(caseModel)
+      val result = Http(s"$serviceUrl/cases")
+        .headers(Seq("Content-Type" -> "application/json")).timeout(5000, 10000)
```
[the remark this answers is not in the document]
Comment (jumpertester) [reply test/it/uk/gov/hmrc/component/CaseSpec.scala]: Reused the expectedCaseBody val.

File: test/it/uk/gov/hmrc/component/data/CaseData.scala [hunk 4]
```diff
@@ -0,0 +1,64 @@
+/*
```
[the remark this answers is not in the document]
Comment (jumpertester) [reply test/it/uk/gov/hmrc/component/data/CaseData.scala]: Removed.

File: test/it/uk/gov/hmrc/component/data/EventData.scala [hunk 5]
```diff
@@ -0,0 +1,41 @@
+/*
```
[the remark this answers is not in the document]
Comment (jumpertester) [reply test/it/uk/gov/hmrc/component/data/EventData.scala]: Removed.

File: build.sbt [hunk 6]
```diff
@@ -24,10 +24,13 @@ lazy val compile = Seq(
 lazy val scope: String = "test,it"
 
 lazy val test = Seq(
+  "com.github.tomakehurst" % "wiremock" % "1.58" % scope,
```
Comment (gokyo) [inline build.sbt]: we should try to use the latest stable version for this dependency

Comment (jumpertester) [reply build.sbt]: Updated wiremock to latest version 2.19.0

**Pull request merged**